### ISSUE TITLE
Stick to the Region Windows in multiple virtual Desktop

### ISSUE THREAD

**Issue by gg31hh** [tom-englert/RegionToShare#64]
First of all, I love "regionToShare" app, I came across it from a YouTuber and this is exactly what I'm looking for. 

However, I found an issue with my usage. I use a laptop and often need to switch between virtual desktops to check my multi-sources notes during a presentation discussion. I noticed, if I were to switch my virtual desktop from Desktop 1 to Desktop 2, The regiontoshare follows along to Desktop2, Thus it auto-share what is that specific region Desktop2 without any boundary indication. Often this will lead to the accidental showing of certain confidential notes opened.

Is there a way to make RegionToShare only share where the app is opened in the particular Windows Virtual Desktop?

Thank you

FYI. im using Windows 11

**Comment by tom-englert** [comment]
Maybe the answer in #54 helps?

**Comment by gg31hh** [comment]
> Maybe the answer in #54 helps?

Yes. at the meantime, I have been doing it like that. just sometime its so difficult when u r in 14" laptop. So i was wondering if anyway to only just share at just ONLY that desktop where regiontoshare app is opened?

**Comment by tom-englert** [comment]
Programmatic control of desktops is very limited, I don't see how this could be achieved 


https://stackoverflow.com/questions/32416843/programmatic-control-of-virtual-desktops-in-windows-10#32417530
